clients/cmd/promtail/main.go

```diff
@@ -20,27 +21,31 @@ import (
 	"github.com/grafana/loki/clients/pkg/logentry/stages"
 	"github.com/grafana/loki/clients/pkg/promtail"
 	"github.com/grafana/loki/clients/pkg/promtail/client"
-	"github.com/grafana/loki/clients/pkg/promtail/config"
-
 	"github.com/grafana/loki/pkg/util"
-	_ "github.com/grafana/loki/pkg/util/build"
 	"github.com/grafana/loki/pkg/util/cfg"
+
+	promtail_config "github.com/grafana/loki/clients/pkg/promtail/config"
+	_ "github.com/grafana/loki/pkg/util/build"
 	util_log "github.com/grafana/loki/pkg/util/log"
 )
 
 func init() {
 	prometheus.MustRegister(version.NewCollector("promtail"))
 }
 
+var mtx sync.Mutex
+
 type Config struct {
-	config.Config   `yaml:",inline"`
-	printVersion    bool
-	printConfig     bool
-	logConfig       bool
-	dryRun          bool
-	configFile      string
-	configExpandEnv bool
-	inspect         bool
+	promtail_config.Config `yaml:",inline"`
+	printVersion           bool
+	printConfig            bool
+	logConfig              bool
+	dryRun                 bool
+	configFile             string
+	configExpandEnv        bool
+	inspect                bool
+
+	cnt int
```

Comment:
What is this for?

Reply:
I have deleted it.
My mistake, this is a debug variable left over.